### PR TITLE
hack: README - add steps for single E2E test

### PR DESCRIPTION
**Pull request by dougsland**
To easily bring new members/developers to the `kpng testing`
efforts.

Signed-off-by: Douglas Schilling Landgraf <dlandgra@redhat.com>

### CODE REVIEW

**Review by nehaLohia27 — APPROVED** [review]

File: hack/README.md [hunk 1]
```diff
@@ -43,6 +43,38 @@ thus the recipe has separate 'functions' for each phase of running KPNG.
...
+
+However, these files can be useful for developers speed up and execute a `single failure test` from the E2E framework.  
+
+The example below focus **only** in the test `should be able to preserve UDP traffic when server pod cycles for a ClusterIP service` and **skip**: `Feature|Federation|PerformanceDNS|Disruptive|Serial|LoadBalancer|KubeProxy|GCE|Netpol|NetworkPolicy` tests.
+
+*<ins>Executing a single KPNG test</ins>*:
+```
+kpng> ./hack/test_e2e.sh -i ipv4 -b iptables
+kpng> cd hack/temp/e2e
+kpng/e2e2>./ginkgo --nodes="25" \
```
Comment (knabben) [inline hack/README.md:66]: is --nodes required? can we put a note of why 25?

File: hack/README.md [hunk 2]
```diff
@@ -43,6 +43,38 @@ thus the recipe has separate 'functions' for each phase of running KPNG.
...
+       --focus="should be able to preserve UDP traffic when server pod cycles for a ClusterIP service" \
+       --skip="Feature|Federation|PerformanceDNS|Disruptive|Serial|LoadBalancer|KubeProxy|GCE|Netpol|NetworkPolicy" \
+        ./e2e.test \
+        -- \
+        --kubeconfig="kubeconfig_tests.conf" \
+        --provider="local" \
+        --dump-logs-on-failure=false \
+        --report-dir="artifacts/reports" \
```
Comment (knabben) [inline hack/README.md:74]: maybe you could explain what is expected in the reports and how to find things.. onliner explaining the file and utility would be great.

File: hack/README.md (hunk 1 above)
Comment (dougsland) [reply hack/README.md:66]: I will add --nodes=1 so in case we have more tests we can easily increase, my bad. good catch.

**Pull request merged**